packages/paste-core/components/alert-dialog/stories/index.stories.tsx

```diff
@@ -228,6 +228,7 @@ export const CustomizedAlertDialog = (): React.ReactElement => {
         ALERT_DIALOG_BODY: {backgroundColor: 'colorBackgroundSuccessWeakest'},
         ALERT_DIALOG_FOOTER: {paddingTop: 'space50', backgroundColor: 'colorBackgroundWarningWeakest'},
       }}
+      disableAnimations={true}
```

Comment:
question: does this still need to be passed with the new `isTestServer` check?

Reply:
Yeah, unfortunately, because CustomizationProvider is basically a new Theme Provider, it resets all the animation stuff. It's a new theme context.

I think what I'm about to regret suggesting I do, is create a Stories Customization Provider which only tables theme and elements. Turns off animations in a single place and also sets getting the current theme in a single location too. Given that is what most of the changes are.

Reply:
Backing out of this idea, it's unfortunate copy pasta from here